backend/utils/auth_utils.py

```
@@ -78,3 +78,8 @@ def validate_date_range(from_date: str, to_date: str, date_format: str = "%Y-%m-
         raise HTTPException(
             status_code=400, detail="From date cannot be after To date."
         )
+
+def clean_guest_email(upn: str) -> str:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)